apps/admin/integration-testing/Makefile

```diff
@@ -27,10 +27,9 @@ build-smartcards:
 build: build-frontend build-backend build-converter-ms-sems build-smartcards
 
 run:
-	export NODE_ENV=production
```

Comment:
This was a fun find - every line of a Makefile is run in a separate shell context. So integration tests haven't actually been running with `NODE_ENV` set to `production`. I fix that here

Reply:
Lol after all that discussion! Nice find